leaderboard/final_html_output.py

```diff
@@ -31,10 +26,25 @@ def final_html_output(df: pd.DataFrame) -> str:
                     },
                 )
             )
-        )
+
+    return html_string
+
+
+def read_file(file_path: str) -> str:
+    """
+    Read file
+
+    Args:
+        file_path (str): The path to the file.
+
+    Returns:
+        str: The contents of the file.
+    """
+    with open(file_path) as file:
+        return file.read()
 
 
-def format_username(username: str):
+def format_username(username: Any) -> str:
```

Comment:
user name type is `str`, why are we changing it to `Any `

Reply:
@Swechhya To make it compatible with `formatters` parameter of `df.to_html()` method. The `formatters` parameter is of type `object`.

![image](https://github.com/user-attachments/assets/5f582bb4-1b1d-4509-9d46-de9b035592dc)
![image](https://github.com/user-attachments/assets/dd7cd9c7-e9ce-410a-9423-d11cabe69574)